packages/react-triple-client-interfaces/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "@titicaca/react-triple-client-interfaces",
+  "version": "5.2.1",
+  "description": "React module for Triple native client interfaces",
+  "main": "lib/index.js",
+  "files": [
+    "lib/*"
+  ],
+  "author": "",
+  "license": "ISC",
```

각 패키지의 package.json을 노멀라이즈 하는 스크립트도 준비중입니다..후후후
```suggestion
  "license": "MIT",
```

---

띠용

---

이걸 못 챙겼었군요 ㅋㅋㅋ

---

ㅠㅠ 이 변경이 package-lock.json의 변경을 또 요구해서 그만 .. 다시 올렸습니다!